### DISCUSSION TITLE
Is there any way to make connectkit work with gnosis safe?

### DISCUSSION THREAD

**Discussion by slavik0329** [family/connectkit#166] [Q&A] [answered]
It seems like theres code built in for this connector, but I am unable to make it work.

**Comment by lochie** [comment, marked as answer]
You can currently use Safe in ConnectKit on mobile devices, but worth noting that there is a [Safe connector](https://wagmi.sh/react/connectors/safe) that we're looking at implementing soon.

![CleanShot —03-22 at 01 13 54](https://user-images.githubusercontent.com/1930210/226633673-24bbdf9b-ea3c-4b20-8b32-6a61a23475ac.png)
